internal/db/db.go

```diff
@@ -28,6 +93,10 @@ type DB interface {
 	InsertCompose(jobId, accountNumber, orgId string, request json.RawMessage) error
 	GetComposes(accountNumber string, limit, offset int) ([]ComposeEntry, int, error)
 	GetCompose(jobId string, accountNumber string) (*ComposeEntry, error)
+	// Returns the count of compose requests performed by an accountNumber since the current date for the duration of (the
+	// sum of) the time intervals. The sum of the time intervals should be negative for the db to search in the past, and
+	// positive to search in the future.
+	CountComposesFor(accountNumber string, intervals ...TimeInterval) (int, error)
```

Comment:
"For" isn't super clear here. `CountComposesSince(accountNumber string, since time.Duration)` reads clearer, can drop the comment then as well.